pandas/plotting/_matplotlib/boxplot.py

```diff
@@ -2,10 +2,13 @@
 import warnings
 
 from matplotlib import pyplot as plt
+from matplotlib.artist import setp
 import numpy as np
 
 from pandas.core.dtypes.generic import ABCSeries
 from pandas.core.dtypes.missing import remove_na_arraylike
+from pandas.core.reshape.concat import concat
```

Comment:
import from pandas

Reply:
Thanks! Just updated it.

Reply:
@jreback I still pretty new here. Can I ask why this place could directly use import from pandas not import as previously ?

Reply:
previously before @datapythonista reorg (just merged)
the plotting code was basically imported during pandas init
and the pandas namespace was not fully defined yet

it’s now essentially decoupled from that so that it a lazy import 
this we can use top of module imports rather than import into each function

Reply:
Appreciated for this detail explanation. This is helpful.